sass/components/Dashboard/capacity.scss

```diff
@@ -29,9 +28,5 @@
 .kubevirt-capacity__graph {
   display: flex;
   flex-direction: column;
-
-  svg {
```

Comment:
@rawagner did you say we needed this overflow?

Reply:
Yes, otherwise tooltips are cut see 
![svg_tooltip](https://user-images.githubusercontent.com/2078045/56640461-5ed80b80-6673-11e9-8f67-3de8852816ae.png)

Reply:
updated